courses/free-courses-en.md

```diff
@@ -736,6 +736,7 @@
 ### Graph Theory
 
 * [Graph Theory](https://youtube.com/playlist?list=PLztBpqftvzxXBhbYxoaZJmnZF6AUQr1mH) - Wrath of Math
+* [Graph Theory](https://nptel.ac.in/courses/111106102) - Prof. Soumen Maity (NPTEL)
```

Comment:
This appears to be a time limited course. We don't list those. Also, we don't include honorifics, such as " Prof."

Reply:
I suggested these changes due to the presence of similar time-limited courses in the repository. We should think about removing them from the repository later.

Reply:
agreed